components/script/dom/htmlscriptelement.rs

```diff
@@ -73,8 +73,8 @@ impl HTMLScriptElement {
             htmlelement:
                 HTMLElement::new_inherited(local_name, prefix, document),
             already_started: Cell::new(false),
-            parser_inserted: Cell::new(creator == ElementCreator::ParserCreated),
-            non_blocking: Cell::new(creator != ElementCreator::ParserCreated),
+            parser_inserted: Cell::new(creator == ElementCreator::ParserCreated(1)),
+            non_blocking: Cell::new(creator != ElementCreator::ParserCreated(1)),
```

Comment:
We should add a new `line_number` member to HTMLScriptElement and initialize it with the line number from the `creator` argument here (we can use 1 in the non-ParserCreated case).